ryu/man/ryu.3

```diff
@@ -0,0 +1,91 @@
+.\" Please update the date every time you change the doc!

```

Comment:
Did you write these by hand or is there a way to auto-generate them from the header files?

Reply:
Wrote it by hand, but the prototypes are definitely being auto-generated somewhere in the world...